src/docs/sphinx/blueprint_mesh.rst

```diff
@@ -392,9 +392,228 @@ To conform, the ``state`` entry must be an *Object* and can have the following o
+
+    conduit::blueprint::mesh::examples::gradient(const std::string &mesh_type, // element type/dimensionality
+                                                 index_t nx,                   // number of grid points along x
+                                                 index_t ny,                   // number of grid points along y
+                                                 index_t nz,                   // number of grid points along z (3d only)
+                                                 Node &res);                   // result container
+
+
+
+The element representation, type, and dimensionality are all configured through the ``mesh_type`` argument. The
+supported values for this parameter and their corresponding effects are outlined in the table below:
+
++---------------+--------------------+-------------------+-------------------+------------------+
+| **Mesh Type** | **Dimensionality** | **Coordset Type** | **Topology Type** | **Element Type** |
++---------------+--------------------+-------------------+-------------------+------------------+
+| uniform       | 2d/3d              | implicit          | implicit          | quad/hex         |
++---------------+--------------------+-------------------+-------------------+------------------+
+|rectilinear    | 2d/3d              | implicit          | implicit          | quad/hex         |
++---------------+--------------------+-------------------+-------------------+------------------+
+|structured     | 2d/3d              | explicit          | implicit          | quad/hex         |
++---------------+--------------------+-------------------+-------------------+------------------+
+|tris           | 2d                 | explicit          | explicit          | tri              |
++---------------+--------------------+-------------------+-------------------+------------------+
+|quads          | 2d                 | explicit          | explicit          | quad             |
++---------------+--------------------+-------------------+-------------------+------------------+
+|tets           | 3d                 | explicit          | explicit          | tet              |
++---------------+--------------------+-------------------+-------------------+------------------+
+|hexs           | 3d                 | explicit          | explicit          | hex              |
++---------------+--------------------+-------------------+-------------------+------------------+
+
+The remainder of this section demonstrates each of the different ``gradient()`` mesh types, outlining
+each type with a simple example that (1) presents the generating call, (2) shows the results of the
+call in Blueprint schema form, and (3) displays the corresponding graphical rendering of this schema.
+
+Uniform
+====================================
+
+* **Call**
+
+.. literalinclude:: ../../tests/docs/t_conduit_docs_blueprint_demos.cpp
+   :lines: 132-137
+   :language: cpp
+   :dedent: 4
+
+* **Schema**
+
```

Comment:
These aren't really Schemas, it's actually the data tree.  Can we just say `Result` instead of Schema ?